repository-data/webfiles/src/main/resources/site/freemarker/common/app-layout-head.ftl

```diff
@@ -15,20 +15,27 @@
     <meta property="og:image:type" content="image/jpeg">
     <meta property="og:image" content="<@hst.webfile path="images/nhs-digital-logo-social.png" fullyQualified=true/>">
     <meta property="og:url" content="${getDocumentUrl()}">
-    
+
     <@hst.headContributions categoryExcludes="htmlBodyEnd, scripts" categoryIncludes="facebookMeta" xhtml=true/>
-    
+
     <!-- Twitter meta tags -->
     <meta name="twitter:card" content="summary_large_image">
     <meta name="twitter:site" content="@NHSDigital">
     <meta name="twitter:image" content="<@hst.webfile path="images/nhs-digital-logo-social.png" fullyQualified=true/>">
     <@hst.headContributions categoryExcludes="htmlBodyEnd, scripts" categoryIncludes="twitterMeta" xhtml=true/>
-    
+
     <#-- Preload fonts to improve performance -->
-    <link rel="preload" href="<@hst.webfile path="fonts/FrutigerLTW01-55Roman.woff2" />" as="font" type="font/woff2" crossorigin>   
-    <link rel="preload" href="<@hst.webfile path="fonts/FrutigerLTW01-56Italic.woff2" />" as="font" type="font/woff2" crossorigin>   
-    <link rel="preload" href="<@hst.webfile path="fonts/FrutigerLTW01-65Bold.woff2" />" as="font" type="font/woff2" crossorigin>   
-    <link rel="preload" href="<@hst.webfile path="fonts/FrutigerLTW01-66BoldItalic.woff2" />" as="font" type="font/woff2" crossorigin>   
+    <link rel="preload" href="<@hst.webfile path="fonts/FrutigerLTW01-55Roman.woff2" />" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="<@hst.webfile path="fonts/FrutigerLTW01-56Italic.woff2" />" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="<@hst.webfile path="fonts/FrutigerLTW01-65Bold.woff2" />" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="<@hst.webfile path="fonts/FrutigerLTW01-66BoldItalic.woff2" />" as="font" type="font/woff2" crossorigin>
```

Comment:
This block all "looks" the same, not sure why git highlights it as a change.

Reply:
Pushing now with jQuery file removed.